common/utils/src/main/resources/error/error-conditions.json

```
@@ -2020,11 +2020,23 @@
     ],
     "sqlState" : "XX000"
   },
-  "INTERVAL_ARITHMETIC_OVERFLOW" : {
-    "message" : [
-      "<message>.<alternative>"
+  "INTERVAL_ARITHMETIC_OVERFLOW": {
+    "message": [
+      "Integer overflow while operating with intervals."
     ],
-    "sqlState" : "22015"
+    "subClass": {
+      "WITH_TRY_SUGGESTION": {
+        "message": [
+          "<message>.<alternative>."
```

I think we can change the structure here to:
```suggestion
          "<alternative>."
```
The other part was just for passing java message

---

Turned out that we couldn't because overflowing while dividing by -1 has fixed message: https://github.com/apache/spark/blob/2fa9f9d48ffb69684e466833d726732233e674b6/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/intervalExpressions.scala#L710